boards/arm/xmc47_relax_kit/xmc47_relax_kit.dts

```diff
@@ -0,0 +1,109 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2023 Schlumberger
+ *
+ */
+
+/dts-v1/;
+
+#include <infineon/xmc4700_F144x2048.dtsi>
+#include <infineon/xmc4700_F144x2048-intc.dtsi>
+#include "xmc47_relax_kit-pinctrl.dtsi"
+
+/ {
+	model = "Infineon XMC4700 Relax Kit board";
+	compatible = "infineon,xmc4700", "infineon,xmc4xxx";
+
+	aliases {
+		led0 = &led1;
+		die-temp0 = &die_temp;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		/* leds are labelled LED1 and LED2 in the relax kit documentation */
+		led1: led1 {
+			gpios = <&gpio5 9 GPIO_ACTIVE_HIGH>;
+		};
+		led2: led2 {
+			gpios = <&gpio5 8 GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
can we add user button as well?

Reply:
![image](https://user-images.githubusercontent.com/46566584/227773487-0c4572b4-19ea-401c-86ec-9f7c5bfbb050.png)

Reply:
Unfortunately P15.13, P15.12 don't support interrupts (there is no ERU connection). And all the samples I've seen with sw0 assume interrupt support. Not sure if it's worth adding it. What do you think?

Reply:
yes, it doesn't make sense to add it.